1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -0,0 +1,8 @@
+
+export function fizzBuzz(number: number) {
+    if (number > 0 && number < 101) {
+        if (number % 15 == 0) return 'FizzBuzz'
+        if (number % 3 == 0) return 'Fizz'
+        if (number % 5 == 0) return 'Buzz'
+    }
```

Comment:
This a cool and simple implementation! You have definitely used the least amount of code possible and done a great job with the tests.

There's a single requirement that's missing, which is in the first part of the assignment description:

"Write a function that takes numbers from 1 to 100 and outputs them as a string".

So in other words, if the `number` is within the valid range, and they're neither multiples or 3 nor 5, we should return the strint version of that `number`.

Also, as a suggestion, I would avoid naming variables/properties the same as TypeScript's keywords (`number: number` in this case) so as to avoid possible TS errors.